cmd/buildinfo/main.go

```diff
@@ -0,0 +1,173 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/google/kf/pkg/kf/describe"
+	"github.com/segmentio/textio"
+)
+
+func main() {
+	uid := flag.Int("uid", 1000, "the user id of the buildpack user")
+	gid := flag.Int("gid", 1000, "the group id of the buildpack user")
+	workspace := flag.String("app", "/workspace", "the workspace for the app")
+	image := flag.String("image", "", "the resulting image")
+	runImage := flag.String("run-image", "", "the image to use as a stack")
+	builderImage := flag.String("builder-image", "", "the image to use as the builder")
+	useCredHelpers := flag.String("use-cred-helpers", "", "whether or not to use cred helpers")
+	cacheVolume := flag.String("cache", "", "the name of the cache volume")
+	buildpackOverrides := flag.String("buildpack", "", "custom buildpacks")
+
+	flag.Parse()
+
+	w := os.Stdout
+
+	describe.SectionWriter(w, "Changing permissions", func(w io.Writer) {
+		for _, dir := range []string{"/builder/home", "/layers", "/cache", *workspace} {
+			fmt.Fprintf(w, "chown -R %d:%d %s\n", *uid, *gid, dir)
+
+			if err := chown(dir, *uid, *gid); err != nil {
```

Comment:
Given that this is called `buildinfo`, I find it surprising it making changes. Can we name this something different?